app/controllers/admin/projects_controller.rb

```diff
@@ -18,7 +18,10 @@ class Admin::ProjectsController < Channels::Admin::BaseController
 
   def index
     respond_to do |format|
-      format.html { collection }
+      format.html do
+        @movable_projects_to_channel = Project.movable_to_channel.order(:name)
```

Comment:
How to you think to name this IV like `@switch_channel_available_projects` or something like that?